### PR TITLE
Update meta descriptions for 25 most popular Learn pages

### PR DESCRIPTION
**Pull request by jwitz**
Part 1 of https://github.com/astronomer/docs/issues/1365

### CODE REVIEW

File: learn/airflow-branch-operator.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 ---
 title: "Branching in Airflow"
 sidebar_label: "Branches"
-description: "Use Apache Airflow's BranchPythonOperator and ShortCircuitOperator to execute conditional branches in your workflow"
 id: airflow-branch-operator
-tags: ["DAGs", "Operators", "Basics", "Tasks"]
 ---
 
+<head>
+  <meta name="description" content="Learn about Airflow’s multiple options for building conditional logic and branching within DAGs, including the BranchPythonOperator and ShortCircuitOperator." />
```
Comment (kwcanuck) [inline learn/airflow-branch-operator.md:8]:
```suggestion
  <meta name="description" content="Learn about the options available in Airflow for building conditional logic and branching within DAGs, including the BranchPythonOperator and ShortCircuitOperator." />
```

File: learn/airflow-branch-operator.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+  <meta name="og:description" content="Learn about Airflow’s multiple options for building conditional logic and branching within DAGs, including the BranchPythonOperator and ShortCircuitOperator." />
```
Comment (kwcanuck) [inline learn/airflow-branch-operator.md:9]:
```suggestion
  <meta name="og:description" content="Learn about the options available in Airflow for building conditional logic and branching within DAGs, including the BranchPythonOperator and ShortCircuitOperator." />
```

File: learn/airflow-branch-operator.md [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
+</head>
+
+
 When designing your data pipelines, you may encounter use cases that require more complex task flows than "Task A > Task B > Task C." For example, you may have a use case where you need to decide between multiple tasks to execute based on the results of an upstream task. Or you may have a case where part of your pipeline should only run under certain external conditions. Fortunately, Airflow has multiple options for building conditional logic and/or branching into your DAGs.
 
 In this guide, we'll cover examples using the `BranchPythonOperator` and `ShortCircuitOperator`, other available branching operators, and additional resources for implementing conditional logic in your Airflow DAGs.
```
Comment (kwcanuck) [inline learn/airflow-branch-operator.md:15]:
```suggestion
In this guide, you'll learn how you can use the `BranchPythonOperator` and `ShortCircuitOperator`, other available branching operators, and additional resources to implement conditional logic in your Airflow DAGs.
```

File: learn/airflow-components.md [hunk 4]
```diff
@@ -1,10 +1,14 @@
 ---
 title: "Airflow components"
 sidebar_label: "Core components"
-description: "Learn about the core components of Apache Airflow."
 id: airflow-components
 ---
 
+<head>
+  <meta name="description" content="Understand the core components of Apache Airflow 2.0+. Review their functions and find out which components to run for specific use cases." />
```
Comment (kwcanuck) [inline learn/airflow-components.md:8]:
```suggestion
  <meta name="description" content="Understand the core components of Apache Airflow 2.0 and later. Review their functions and find out which components to run for specific use cases." />
```

File: learn/airflow-components.md [hunk 5]
```diff
@@ -1,10 +1,14 @@
...
+  <meta name="og:description" content="Understand the core components of Apache Airflow 2.0+. Review their functions and find out which components to run for specific use cases." />
```
Comment (kwcanuck) [inline learn/airflow-components.md:9]:
```suggestion
  <meta name="og:description" content="Understand the core components of Apache Airflow 2.0 and later. Review their functions and find out which components to run for specific use cases." />
```

File: learn/airflow-components.md [hunk 6]
```diff
@@ -1,10 +1,14 @@
...
+</head>
+
 When working with Airflow, understanding the underlying infrastructure components and how they function can help you develop and run your DAGs, troubleshoot issues, and successfully run Airflow.
 
 In this guide, we'll describe the core components of Airflow and managing Airflow infrastructure for high availability. Some of the components and features described in this topic are unavailable in earlier Airflow versions.
```
Comment (kwcanuck) [inline learn/airflow-components.md:14]:
```suggestion
In this guide, you'll learn about the core components of Airflow and how to manage Airflow infrastructure for high availability. Some of the components and features described in this topic are unavailable in earlier Airflow versions.
```

File: learn/airflow-dbt.md [hunk 7]
```diff
@@ -1,11 +1,14 @@
...
 title: "Orchestrate dbt with Airflow"
 sidebar_label: "dbt"
-description: "Orchestrate dbt models with your Airflow DAGs."
 id: airflow-dbt
-keywords: [DAGs, Integrations]
 ---
 
+<head>
+  <meta name="description" content="See how to use the dbt Cloud Provider to orchestrate dbt Cloud with Airflow. Learn use cases for orchestrating dbt Core using the BashOperator." />
+  <meta name="og:description" content="See how to use the dbt Cloud Provider to orchestrate dbt Cloud with Airflow. Learn use cases for orchestrating dbt Core using the BashOperator." />
```
Comment (kwcanuck) [inline learn/airflow-dbt.md:9]:
```suggestion
  <meta name="og:description" content="Learn how to use the dbt Cloud Provider to orchestrate dbt Cloud with Airflow. Learn use cases for orchestrating dbt Core using the BashOperator." />
```

File: learn/airflow-passing-data-between-tasks.md [hunk 8]
```diff
@@ -1,11 +1,14 @@
 ---
 title: "Passing data between Airflow tasks"
 sidebar_label: "Passing data between tasks"
-description: "Learn about methods available for sharing metadata and information between tasks in your Apache Airflow DAGs, including XCom."
 id: airflow-passing-data-between-tasks
-tags: [DAGs, XCom, Tasks, Dependencies]
 ---
 
+<head>
+  <meta name="description" content="Learn more about the most commonly used methods to implement data sharing between your Airflow tasks, including an in-depth explanation of XCom." />
```
Comment (kwcanuck) [inline learn/airflow-passing-data-between-tasks.md:8]:
```suggestion
  <meta name="description" content="Learn more about the most common methods to implement data sharing between your Airflow tasks, including an in-depth explanation of XCom." />
```

File: learn/airflow-passing-data-between-tasks.md [hunk 9]
```diff
@@ -1,11 +1,14 @@
...
+  <meta name="og:description" content="Learn more about the most commonly used methods to implement data sharing between your Airflow tasks, including an in-depth explanation of XCom." />
```
Comment (kwcanuck) [inline learn/airflow-passing-data-between-tasks.md:9]:
```suggestion
  <meta name="og:description" content="Learn more about the most common methods to implement data sharing between your Airflow tasks, including an in-depth explanation of XCom." />
```

File: learn/airflow-scaling-workers.md [hunk 10]
```diff
@@ -1,11 +1,14 @@
...
 title: "Scaling Airflow to optimize performance"
 sidebar_label: "Scaling Airflow"
-description: "How to scale your Airflow environment."
 id: airflow-scaling-workers
-tags: [Workers, Concurrency, Parallelism, DAGs]
 ---
 
+<head>
+  <meta name="description" content="See which parameters to modify when scaling up data pipelines to make the most of Airflow. Learn about the environment, DAG, and task-level settings." />
+  <meta name="og:description" content="See which parameters to modify when scaling up data pipelines to make the most of Airflow. Learn about the environment, DAG, and task-level settings." />
```
Comment (kwcanuck) [inline learn/airflow-scaling-workers.md:9]:
```suggestion
  <meta name="og:description" content="Learn which parameters to modify when scaling up data pipelines to make the most of Airflow. Learn about the environment, DAG, and task-level settings." />
```

File: learn/airflow-sql.md [hunk 11]
```diff
@@ -1,10 +1,15 @@
 ---
 title: "Using Airflow to Execute SQL"
-description: "Execute queries, parameterize queries, and embed SQL-driven ETL in Apache Airflow DAGs."
 id: airflow-sql
 sidebar_label: "Run SQL"
 ---
 
+<head>
+  <meta name="description" content="Learn the best practices for executing SQL from your DAG. Get to know Airflow’s SQL-related operators and see how to use Airflow for common SQL use cases." />
```
Comment (kwcanuck) [inline learn/airflow-sql.md:8]:
```suggestion
  <meta name="description" content="Learn the best practices for executing SQL from your DAG. Get to know the Airflow SQL-related operators and learn how to use Airflow for common SQL use cases." />
```

File: learn/airflow-sql.md [hunk 12]
```diff
@@ -1,10 +1,15 @@
...
+  <meta name="og:description" content="Learn the best practices for executing SQL from your DAG. Get to know Airflow’s SQL-related operators and see how to use Airflow for common SQL use cases." />
```
Comment (kwcanuck) [inline learn/airflow-sql.md:9]:
```suggestion
  <meta name="og:description" content="Learn the best practices for executing SQL from your DAG. Learn about Airflow SQL-related operators and how to use Airflow for common SQL use cases." />
```

File: learn/airflow-ui.md [hunk 13]
```diff
@@ -5,6 +5,11 @@ description: "An overview of the Airflow UI"
 id: airflow-ui
 ---
 
+<head>
+  <meta name="description" content="Explore the Airflow UI, which helps with monitoring and troubleshooting your data pipelines. See some of its key features and visualizations." />
```
Comment (kwcanuck) [inline learn/airflow-ui.md:9]:
```suggestion
  <meta name="description" content="Explore the Airflow UI, which helps you monitor and troubleshoot your data pipelines. Learn about some of its key features and visualizations." />
```

File: learn/airflow-ui.md [hunk 14]
```diff
@@ -5,6 +5,11 @@ description: "An overview of the Airflow UI"
...
+  <meta name="og:description" content="Explore the Airflow UI, which helps with monitoring and troubleshooting your data pipelines. See some of its key features and visualizations." />
```
Comment (kwcanuck) [inline learn/airflow-ui.md:10]:
```suggestion
  <meta name="og:description" content="Explore the Airflow UI, which helps you monitor and troubleshoot your data pipelines. Learn about some of its key features and visualizations." />
```

File: learn/bashoperator.md [hunk 15]
```diff
@@ -5,6 +5,11 @@ id: bashoperator
 sidebar_label: BashOperator
 ---
 
+<head>
+  <meta name="description" content="Learn how to use the BashOperator to run bash commands and bash scripts. See an example of how to run scripts in other languages than Python." />
```
Comment (kwcanuck) [inline learn/bashoperator.md:9]:
```suggestion
  <meta name="description" content="Learn how to use the BashOperator to run bash commands and bash scripts. Review examples of how to run scripts in other languages than Python." />
```

File: learn/bashoperator.md [hunk 16]
```diff
@@ -5,6 +5,11 @@ id: bashoperator
...
+  <meta name="og:description" content="Learn how to use the BashOperator to run bash commands and bash scripts. See an example of how to run scripts in other languages than Python." />
```
Comment (kwcanuck) [inline learn/bashoperator.md:10]:
```suggestion
  <meta name="og:description" content="Learn how to use the BashOperator to run bash commands and bash scripts. Review examples of how to run scripts in other languages than Python." />
```

File: learn/connections.md [hunk 17]
```diff
@@ -1,10 +1,14 @@
 ---
 title: "Manage connections in Apache Airflow"
 sidebar_label: "Connections"
-description: "An overview of how connections work in the Airflow UI."
 id: connections
 ---
 
+<head>
+  <meta name="description" content="View tips on how to set up, manage, and maintain different types of connections in Apache Airflow. Use example connection configurations as the basis for your own connections." />
```
Comment (kwcanuck) [inline learn/connections.md:8]:
```suggestion
  <meta name="description" content="Learn how to set up, manage, and maintain different types of connections in Apache Airflow. Use example connection configurations as the basis for your own connections." />
```

File: learn/connections.md [hunk 18]
```diff
@@ -1,10 +1,14 @@
...
+  <meta name="og:description" content="View tips on how to set up, manage, and maintain different types of connections in Apache Airflow. Use example connection configurations as the basis for your own connections." />
```
Comment (kwcanuck) [inline learn/connections.md:9]:
```suggestion
  <meta name="og:description" content="Learn how to set up, manage, and maintain different types of connections in Apache Airflow. Use example connection configurations as the basis for your own connections." />
```

File: learn/dynamically-generating-dags.md [hunk 19]
```diff
@@ -1,10 +1,14 @@
 ---
 title: "Dynamically generating DAGs in Airflow"
 sidebar_label: "Dynamically generate DAGs"
-description: "Use a base DAG template to create multiple DAGs."
 id: dynamically-generating-dags
 ---
 
+<head>
+  <meta name="description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. See examples of how to generate DAGs using single- and multiple-file methods." />
```
Comment (kwcanuck) [inline learn/dynamically-generating-dags.md:8]:
```suggestion
  <meta name="description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. Use examples to generate DAGs using single- and multiple-file methods." />
```

File: learn/dynamically-generating-dags.md [hunk 20]
```diff
@@ -1,10 +1,14 @@
...
+  <meta name="og:description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. See examples of how to generate DAGs using single- and multiple-file methods." />
```
Comment (kwcanuck) [inline learn/dynamically-generating-dags.md:9]:
```suggestion
  <meta name="og:description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. Use examples to generate DAGs using single- and multiple-file methods." />
```

File: learn/error-notifications-in-airflow.md [hunk 21]
```diff
@@ -1,10 +1,14 @@
 ---
 title: "Manage Airflow DAG notifications"
 sidebar_label: "DAG alerts"
-description: "Methods for managing notifications in your Airflow DAGs."
 id: error-notifications-in-airflow
 ---
 
+<head>
+  <meta name="description" content="Master the basics of Apache Airflow notifications. Find out how to set up automatic email and Slack notifications to be alerted of events in your DAGs." />
```
Comment (kwcanuck) [inline learn/error-notifications-in-airflow.md:8]:
```suggestion
  <meta name="description" content="Master the basics of Apache Airflow notifications. Learn how to set up automatic email and Slack notifications to be alerted of events in your DAGs." />
```

File: learn/error-notifications-in-airflow.md [hunk 22]
```diff
@@ -1,10 +1,14 @@
...
+  <meta name="og:description" content="Master the basics of Apache Airflow notifications. Find out how to set up automatic email and Slack notifications to be alerted of events in your DAGs." />
```
Comment (kwcanuck) [inline learn/error-notifications-in-airflow.md:9]:
```suggestion
  <meta name="og:description" content="Master the basics of Apache Airflow notifications. Learn how to set up automatic email and Slack notifications to be alerted of events in your DAGs." />
```

File: learn/task-groups.md [hunk 23]
```diff
@@ -1,10 +1,14 @@
 ---
 title: "Airflow task groups"
 sidebar_label: "Task groups"
-description: "Use task groups to build modular Airflow workflows."
 id: task-groups
 ---
 
+<head>
+  <meta name="description" content="Follow Astronomer’s step-by-step guide to to use task groups for organizing tasks within the graph view of the Airflow user interface." />
```
Comment (kwcanuck) [inline learn/task-groups.md:8]:
```suggestion
  <meta name="description" content="Learn how you can use task groups to organize tasks within the graph view of the Airflow user interface." />
```
If you don't use this recommendation, update "to to".

File: learn/task-groups.md [hunk 24]
```diff
@@ -1,10 +1,14 @@
...
+  <meta name="og:description" content="Follow Astronomer’s step-by-step guide to to use task groups for organizing tasks within the graph view of the Airflow user interface." />
```
Comment (kwcanuck) [inline learn/task-groups.md:9]:
```suggestion
  <meta name="description" content="Learn how you can use task groups to organize tasks within the graph view of the Airflow user interface." />
```
If you don't use this recommendation, update "to to".

File: learn/templating.md [hunk 25]
```diff
@@ -1,10 +1,15 @@
 ---
 title: "Use Airflow templates"
 sidebar_label: "Jinja templates"
-description: "How to leverage the power of Jinja templating when writing your DAGs."
 id: templating
 ---
 
+<head>
+  <meta name="description" content="Learn about Jinja templating in Apache Airflow and see examples of how to pass dynamic information into task instances at runtime. " />
```
Comment (kwcanuck) [inline learn/templating.md:8]:
```suggestion
  <meta name="description" content="Learn about Jinja templating in Apache Airflow and how to pass dynamic information into task instances at runtime. " />
```

File: learn/templating.md [hunk 26]
```diff
@@ -1,10 +1,15 @@
...
+  <meta name="og:description" content="Learn about Jinja templating in Apache Airflow and see examples of how to pass dynamic information into task instances at runtime. " />
```
Comment (kwcanuck) [inline learn/templating.md:9]:
```suggestion
  <meta name="og:description" content="Learn about Jinja templating in Apache Airflow and how to pass dynamic information into task instances at runtime. " />
```

File: learn/what-is-a-hook.md [hunk 27]
```diff
@@ -1,10 +1,15 @@
 ---
 title: "Airflow hooks"
 sidebar_label: "Hooks"
-description: "An introduction to hooks in Apache Airflow."
 id: what-is-a-hook
 ---
 
+<head>
+  <meta name="description" content="Learn about hooks and general guidelines on using them in Apache Airflow. See an example of implementing two different hooks in a DAG." />
```
Comment (kwcanuck) [inline learn/what-is-a-hook.md:8]:
```suggestion
  <meta name="description" content="Learn about hooks and how they should be used in Apache Airflow. See an example of implementing two different hooks in a DAG." />
```

File: learn/what-is-a-hook.md [hunk 28]
```diff
@@ -1,10 +1,15 @@
...
+  <meta name="og:description" content="Learn about hooks and general guidelines on using them in Apache Airflow. See an example of implementing two different hooks in a DAG." />
```
Comment (kwcanuck) [inline learn/what-is-a-hook.md:9]:
```suggestion
  <meta name="og:description" content="Learn about hooks and how they should be used in Apache Airflow. See an example of implementing two different hooks in a DAG." />
```

**Review by kwcanuck — APPROVED** [review]

File: learn/task-groups.md (hunk 23 above)
Comment (jwitz) [inline learn/task-groups.md:8]:
```suggestion
  <meta name="description" content="Follow Astronomer’s step-by-step guide to use task groups for organizing tasks within the graph view of the Airflow user interface." />
```

File: learn/task-groups.md (hunk 24 above)
Comment (jwitz) [inline learn/task-groups.md:9]:
```suggestion
  <meta name="og:description" content="Follow Astronomer’s step-by-step guide to use task groups for organizing tasks within the graph view of the Airflow user interface." />
```

**Pull request merged**